vm/src/obj/objdict.rs

```diff
@@ -146,6 +146,15 @@ impl PyDictRef {
         }
     }
 
+    fn ne(self, other: PyObjectRef, vm: &VirtualMachine) -> PyResult {
+        if let Some(other) = other.payload::<PyDict>() {
+            let eq = !self.inner_eq(other, vm)?;
```

Comment:
This local variable holds a not-equals result, thus calling it `eq` seems misleading to me. I’d either rename it to `neq` *or* change it to
```rust
let eq = self.inner_eeq(other, vm)?;
Ok(vm.ctx.new_bool(!eq))
```

Reply:
Sure. Lazy copy-pasting happened here. Will fix.